api/src/registrations.py

```diff
@@ -236,6 +239,30 @@ def rsvp_view():
 
     return jsonify({"rsvpList":rsvpList, "allList": allList}),200
 
+
+@registrations_api.route('/rsvp/status', methods = ['GET'])
+@jwt_required
+def rsvp_status():
+
+    """For a user, check if he/she RSVPed or not
+
+    :reqheader Authorization: ``Bearer <JWT Token>``
+
+    :status 200: Successful
+    """
+
+    id = get_jwt_identity()
+    user = db.users.find_one({'_id' : ObjectId(id)})
+
+    eventInfo = user["registrations"][0] # only need to check first one, because there should only be one entry
+    if("rsvp" not in eventInfo or eventInfo["rsvp"] == False):
+        return jsonify({"status":False}),200 
```

Comment:
making the response message more descriptive is a better practice if we have time